### ISSUE TITLE
Shopify integration

### ISSUE THREAD

**Issue by smity81435** [airbnb/lottie#115]
Hey Everyone,
I am trying to integrate two Lottie animations (img sequences) into my Shopify theme, but I am having some issues. 
So far:
1. I've got the JSON file loading as animationData in the params 
2. I've got the images in assets/images/...
3.  I've got a custom script calling the lottie animation in assets/animationIndex.js
4. I've got a console log just before the lottie call confirming that we are getting into the call.
5. I've got the div with a class of lottie (all scss matching the demo)

And then... nothing. No errors, no animation or even 404's. Just blank. 

My Theory: I think it has something to do with the way that liquid retrieves assets. It's not retrieving the img files that make up the animation, and so it is not running. By why wouldn't it say: img not found. or something like that? There isn't exactly a way for me to call an entire asset folder in liquid, and even if there was, how would I pass that to Lottie?
Im stumped. Does anyone have experience using Lottie img sequences in Shopify?

**Comment by smity81435** [comment]
Ok so after talking with the Shopify Plus support staff it basically all came down to the facts that :
1. Shopify doesn't allow uploading custom image folders, only individual files.
2. Because of the JSON structure, that isnt going to work with liquid asset tags.
3. Shopify optimizes all of the images uploaded to their serves, then serves them back out with their CDN. This actually changes the name of the images with a mild hashing. (i.e. img01.png becomes like ad3@12e1df4$4235%5.png)
4. It MIGHT be possible to for the lottie library and convert the JSON parser to take shopify filters, but this is a hard maybe.
TLDR: Shopify is not compatible with Lottie PNG Sequences. =[

**Comment by smity81435** [comment]
Ok So, Update on this... 
You actually can make it work. Here are the steps:
1. Upload all of your images to an S3 bucket.
2. In Lottie, specifically in the function getAssetsPath, you need to modify that to accept a more generic source url. 
3. Either Host your JSON and Lottie on S3 or place them in Liquid filters at the bottom of the theme.liquid body.
If you need more specific guidance on this shoot me a message. Perhaps we could accept absolute URL's in the img path for this?

**Comment by bodymovin** [comment]
Hi, does this help for this scenario?
https://github.com/airbnb/lottie-web/wiki/loadAnimation-options#assetspath
as long as the assets names are the same, it should work. without modifying the json data.

**Comment by gpeal** [comment]
**Closing all library issues on this repo**
This repo is just for the docs on [airbnb.io/lottie](https://airbnb.io/lottie)

I would bring this up with Shopify on whatever community or support forum they have.
